software-peer-review-guide/author-guide.md

```diff
@@ -1,105 +1,104 @@
+package. You may get feedback from an editor and two reviewers. Changes could 
+take time. Please consider this before submitting to us. You can read more about the timeline to make changes in our [peer review policies page](../about-peer-review/policies-guidelines).
+2. Peer review is lead by a diverse group of volunteer editors and reviewers. 
+Please be considerate when engaging with everyone online. 
+```
+
 ## 1. Do you plan to continue to maintain your package?
 One of the goals of pyOpenSci is to maintain a curated list of 
 community-approved, maintained and vetted tools that support open science workflows. 
 
-As such we review packages that will be useful to the community
+As such, we review packages that will be useful to the community
 and maintained over time. While we understand that burnout is real,
 and you may move on in the future to other projects, we ask that you commit
-to maintaining your package for at least 1-2 years. 
+to maintaining your package for at least 1-2 years after the review process 
+is complete. 
 
 If that maintenance commitment is too much, you might consider submitting 
-your package to another Journal.
+your package to a Journal that is more focused on publication only.
 
-If you need to step from maintaining your package, please let us know
-in advance. We will try to help you either.
-
-* Find a new maintainer to take over your project or
-* Sunset your package.
 
-If the package is sunsetted we will remove it from our curated list 
-of vetted tools. 
 
 ### Who should submit the package for review?
 
 If you have a team of people maintaining your package, please be sure
-that the submitting author is the person who "owns" or leads that maintenance. That person will become the long term point of contact 
-for pyOpenSci.
+that the submitting author is the person who "owns" or leads that maintenance. 
+That person will become the long term point of contact 
+for pyOpenSci. 
+* Please also include the names of all maintainers on the project
+as we also want to ensure that everyone working on the project gets full credit 
+for their effort. 
+
 
 ```{note}
 If your package is more of a tool to support a specific workflow that 
-either may not be maintained long term or that may be so specific that 
-it won't have a broad user base, you might consider submitting it 
-directly to the Journal of Open Source Software (JOSS). 
+either:
+* may not be maintained long term or 
+* that may be so specific that it won't have a user base outside of your lab or work environment, 
+
+please consider submitting it directly to a publisher like the
+Journal of Open Source Software (JOSS). A publisher like JOSS has less
+emphasis on long term software maintenance and focuses more on 
+publication quality and citation / credit.  
 ```
 
 
 ## 2. Does Your Package Meet Packaging Requirements?
 Before submitting your project for review with pyOpenSci, make sure that it
```

Comment:
we should probably include the editor checks in this page as well

https://output.circle-artifacts.com/output/job/e94ea88d-ec44-4001-93a2-a44af96c79ea/artifacts/0/html/software-peer-review-guide/author-guide.html

so the author can see the checks

Reply:
i like that idea!

Reply:
fixed